Client/Frontend/Home/RecentTabsPanel.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+import Shared
+import Storage
+import XCGLogger
+import Deferred
+
+private let log = Logger.browserLogger
+
+class RecentTabsPanel: UIViewController, HomePanel {
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    var profile: Profile!
```

Comment:
Nit: Instead of `Profile!`, we can clean this up a bit by creating a custom `init`method that takes the profile as a parameter. This would change this to `let profile: Profile` instead so you don't have to worry about the `!?!?!`